src/main/java/de/tum/in/www1/artemis/web/rest/programming/ProgrammingExerciseResource.java

```diff
@@ -729,7 +729,7 @@ public ResponseEntity<Void> reset(@PathVariable Long exerciseId, @RequestBody Pr
     @EnforceAtLeastEditor
     @FeatureToggle(Feature.ProgrammingExercises)
     public ResponseEntity<ProgrammingExercise> reEvaluateAndUpdateProgrammingExercise(@PathVariable long exerciseId, @RequestBody ProgrammingExercise programmingExercise,
-            @RequestParam(value = "deleteFeedback", required = false) Boolean deleteFeedbackAfterGradingInstructionUpdate) throws JsonProcessingException {
+            @RequestParam(value = "deleteFeedback", required = false, defaultValue = "false") boolean deleteFeedbackAfterGradingInstructionUpdate) throws JsonProcessingException {
```

Comment:
if you provide a default value you can get rid of the required annotation
```suggestion
            @RequestParam(value = "deleteFeedback", defaultValue = "false") boolean deleteFeedbackAfterGradingInstructionUpdate) throws JsonProcessingException {
```

Reply:
done.